node/node.go

```diff
@@ -256,8 +258,6 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, db ethdb.Database, is
 			os.Exit(1)
 		}
 		node.blockchain = chain
-		// Store the genesis shard state into db.
-		node.blockchain.StoreNewShardState(node.blockchain.CurrentBlock(), nil)
```

Comment:
why this is removed?

Reply:
It's not removed. it's moved to the other method InitGenesisShardState